packages/pds/tests/account-deletion.test.ts

```diff
@@ -135,49 +136,53 @@ describe('account deletion', () => {
   })
 
   it('no longer store the user account or repo', async () => {
-    updatedDbContents = await getDbContents(db)
-    expect(updatedDbContents.roots).toEqual(
-      initialDbContents.roots.filter((row) => row.did !== carol.did),
+    updatedDbContents = await getDbContents(ctx)
+    expect(updatedDbContents.repoRoots).toEqual(
+      initialDbContents.repoRoots.filter((row) => row.did !== carol.did),
     )
-    expect(updatedDbContents.users).toEqual(
-      initialDbContents.users.filter((row) => row.did !== carol.did),
-    )
-    expect(updatedDbContents.blocks).toEqual(
-      initialDbContents.blocks.filter((row) => row.creator !== carol.did),
+    expect(updatedDbContents.userAccounts).toEqual(
+      initialDbContents.userAccounts.filter((row) => row.did !== carol.did),
     )
     // check all seqs for this did are gone, except for the tombstone
     expect(
-      updatedDbContents.seqs.filter((row) => row.eventType !== 'tombstone'),
-    ).toEqual(initialDbContents.seqs.filter((row) => row.did !== carol.did))
+      updatedDbContents.repoSeqs.filter((row) => row.eventType !== 'tombstone'),
+    ).toEqual(initialDbContents.repoSeqs.filter((row) => row.did !== carol.did))
     // check we do have a tombstone for this did
     expect(
-      updatedDbContents.seqs.filter(
+      updatedDbContents.repoSeqs.filter(
         (row) => row.did === carol.did && row.eventType === 'tombstone',
       ).length,
     ).toEqual(1)
-
-    expect(updatedDbContents.records).toEqual(
-      initialDbContents.records.filter((row) => row.did !== carol.did),
+    expect(updatedDbContents.appPasswords).toEqual(
+      initialDbContents.appPasswords.filter((row) => row.did !== carol.did),
+    )
+    expect(updatedDbContents.emailTokens).toEqual(
+      initialDbContents.emailTokens.filter((row) => row.did !== carol.did),
+    )
+    expect(updatedDbContents.refreshTokens).toEqual(
+      initialDbContents.refreshTokens.filter((row) => row.did !== carol.did),
     )
   })
 
+  it('deletes the users actor store', async () => {
+    const sqliteDir = network.pds.ctx.cfg.db.directory
+    const dbExists = await fileExists(path.join(sqliteDir, carol.did))
+    expect(dbExists).toBe(false)
+    const walExists = await fileExists(path.join(sqliteDir, `${carol.did}-wal`))
+    expect(walExists).toBe(false)
+    const shmExists = await fileExists(path.join(sqliteDir, `${carol.did}-shm`))
+    expect(shmExists).toBe(false)
+  })
```

Comment:
If I understand correctly these may not exist because the actual file lives at `${sqliteDir}/{hash}/${did}...`.

Reply:
yup missed this update 😅 

fixed it when i updated the fs location a few min ago 👌